crates/database/src/orders.rs

```diff
@@ -795,6 +774,34 @@ pub async fn user_orders_with_quote(
         .await
 }
 
+#[derive(Debug, PartialEq, Eq, sqlx::FromRow)]
+pub struct DbOrderUid(pub OrderUid);
+
+pub fn updated_order_uids_after(
+    ex: &mut PgConnection,
+    after_block: i64,
+) -> BoxStream<'_, Result<DbOrderUid, sqlx::Error>> {
+    const QUERY: &str = r#"
+SELECT DISTINCT order_uid FROM (
+    SELECT order_uid FROM trades WHERE block_number > $1
+    UNION
+    SELECT order_uid FROM order_execution WHERE block_number > $1
```

Comment:
`order_execution` not actually an input for solvable orders so I'd rather remove it.

Reply:
It is used to calculate executed surplus fee: https://github.com/cowprotocol/services/blob/267f55059464a798cf5f75a055d3c09fb85371b9/crates/database/src/orders.rs#L551

Reply:
But this is not related to this query in this function right?

Reply:
The `surplus_fee` is technically irrelevant for the `SolvableOrdersCache` but from the DB's perspective this logic is used to determine which orders have any data of the `FullOrder` changed so I'd say since the query would be still very vast we should keep this to avoid inconsistent state. (i.e. caching is hard and we should not cut corners unnecessarily)